sidebars.js

```diff
@@ -69,6 +69,9 @@ module.exports = {
                     label: "Multi-cluster Failover",
                     items: [
                         "userguide/failover/failover",
```

Comment:
Do we need to delete the original article?

Reply:
Yes, I plan to do it.

Reply:
Updated, please take a look.